xbmc/network/httprequesthandler/HTTPVfsHandler.cpp

```diff
@@ -67,6 +70,19 @@ CHTTPVfsHandler::CHTTPVfsHandler(const HTTPRequest &request)
             }
           }
         }
+
+        // Check auto-mounted sources
```

Comment:
Maybe also add a comment further up that we are first checking configured sources.

Reply:
Done